gettsim/data/eink_st.yaml

```diff
@@ -1,1025 +1,175 @@
-  name:
-    de:
-    en:
-  description:
-    de:
-    en:
-  values:
-    1984-01-01:
-      value: 10000
-      note:
-    1996-01-01:
-      value: 10000
-      note: .
-t_yden3:
-  name:
-    de:
-    en:
-  description:
-    de:
-    en:
-  values:
-    1999-01-01:
-      value: 10000
-      note:
-t_ynom1:
-  name:
-    de:
-    en:
-  description:
-    de:
-    en:
-  values:
-    1984-01-01:
-      value: 18000
-      note:
-    1988-01-01:
-      value: 17982
-      note: .
-    1990-01-01:
-      value: 8100
-      note: .
-    1996-01-01:
-      value: 12042
-      note: .
-    1998-01-01:
-      value: 12312
-      note:
-    1999-01-01:
-      value: 13014
-      note:
-    2000-01-01:
-      value: 13446
-      note: .
-    2001-01-01:
-      value: 14040
-      note: .
-t_ynom2:
-  name:
-    de:
-    en:
-  description:
-    de:
-    en:
-  values:
-    1984-01-01:
-      value: 60000
-      note:
-    1986-01-01:
-      value: 80000
-      note:
-    1988-01-01:
-      value: 79974
-      note: .
-    1996-01-01:
-      value: 55674
-      note: .
-    1998-01-01:
-      value: 58590
-      note:
-    1999-01-01:
-      value: 17010
-      note:
-    2000-01-01:
-      value: 17442
-      note: .
-    2001-01-01:
-      value: 18036
-      note: .
-t_ynom3:
-  name:
-    de:
-    en:
-  description:
-    de:
-    en:
-  values:
-    1999-01-01:
-      value: 66312
-      note:
+  note:
+    de: Der quadratische Anstieg wird nach der Formel des Progressionsfaktors von gettsim berechnet. Er bewirkt, dass der Grenzsteuersatz linear ansteigt und so sprunghafte Übergänge zwischen den Tarifzonen vermieden werden
+    en: The quadratic rate is calculated by gettsim with the formula of the Progressionsfaktor. He guratnees that the marginal tax rate increases linear and there are no jumps between income sections.
+  type: piecewise_quadratic
+  progressionsfaktor: True
+  2002-01-01:
+    0:
+      lower_threshold: 0
+      upper_threshold: 7235
+      rate_linear: 0
+      rate_quadratic: 0
+      intercept_at_lower_threshold: 0
+    1:
+      upper_threshold: 9251
+      rate_linear: 0.199
+    2:
+      upper_threshold: 55007
+      rate_linear: 0.23
+    3:
+      upper_threshold: inf
+      rate_linear: 0.485
+      rate_quadratic: 0
+  2004-01-01:
+    0:
+      lower_threshold: 0
+      upper_threshold: 7664
+      rate_linear: 0
+      intercept_at_lower_threshold: 0
+    1:
+      upper_threshold: 12739
+      rate_linear: 0.16
+    2:
+      upper_threshold: 52151
+      rate_linear: 0.2405
+    3:
+      upper_threshold: inf
+      rate_linear: 0.45
+      rate_quadratic: 0
+  2005-01-01:
+    0:
+      lower_threshold: 0
+      upper_threshold: 7664
+      rate_linear: 0
+      rate_quadratic: 0
+      intercept_at_lower_threshold: 0
+    1:
+      upper_threshold: 12739
+      rate_linear: 0.15
+    2:
+      upper_threshold: 52151
+      rate_linear: 0.2397
+    3:
+      upper_threshold: inf
+      rate_linear: 0.42
```

Comment:
Can't we just drop this threshold?

Reply:
Unfortnunatly not, as we need to specify that this rule is not capped! HMG said to specify this explicit and with implicit leaving it out.

Reply:
Yeah, better explicit than implicit. I agree that for upper thresholds, I do not see much of a use case, but for lower thresholds leaving 0 or -inf implicit is often dangerous. Then rather require it everywhere.